packages/app/src/layouts/default.vue

```diff
@@ -15,6 +15,7 @@
       :page-name="currentRoute.name?.toString()"
       data-cy="app-header-bar"
       :allow-automatic-prompt-open="true"
+      @header-loaded="handleHeaderLoaded"
```

Comment:
See comment in `HeaderBar`.  The problem this is currently solving can be encapsulated in the `HeaderBar`

Reply:
Done!